src/testRunner/unittests/tsc/runWithoutArgs.ts

```diff
@@ -14,5 +14,14 @@ namespace ts {
             fs: () => loadProjectFromFiles({}),
             commandLineArgs: [],
         });
+
+        verifyTsc({

```

Comment:
Seems like these tests generate a `.js` file - would be awesome if we could change that though.